woocommerce/woocommerce
# Remove PHP 8.1 warnings and errors

### All Submissions:






### Changes proposed in this Pull Request:



Fixes code that causes PHP 8.1 warnings and errors.

Ad ini_set: Based on this source https://php.watch/versions/8.1/auto_detect_line_endings-ini-deprecated and wiki article https://en.wikipedia.org/wiki/Newline, it seems that it's safe to remove the call to automatically detect line endings, as it only helps with `\r`, which shouldn't really be used on any modern OS.

Removing the `mysql_*` call, since our minimum supported PHP version is 7.2 and this was removed in PHP 7.0 (and has been deprecated since PHP 5.5).

Internal ref: p7bje6-2wT-p2

### How to test the changes in this Pull Request:

1. Ensure the db information is still displayed in the System Status Report
2. Ensure that tax import still works (e.g. add a couple of tax settings, export tax settings, delete settings, import tax settings back)


### Other information:




### FOR PR REVIEWER ONLY:
